src/Agent/NewRelic/Agent/Extensions/Providers/Wrapper/MicrosoftExtensionsLogging/MicrosoftLoggingWrapper.cs

```diff
@@ -28,7 +28,11 @@ public class MicrosoftLoggingWrapper : IWrapper
 
         public CanWrapResponse CanWrap(InstrumentedMethodInfo methodInfo)
         {
-            return new CanWrapResponse(WrapperName.Equals(methodInfo.RequestedWrapperName));
+            if (!LogProviders.KnownLogProvider)
```

Comment:
This might read clearer if `KnownLogProvider` was changed to something like `KnownLogProviderEnabled` or `KnownMELProviderEnabled`. Renaming this field would also make it easier to see the difference between that field and the list of known providers `KnownLogProviders`.

Reply:
Sure, that makes sense.